### PR TITLE
consensus: fix OPEN_CONSENSUS mode

### PR DESCRIPTION
**Pull request by herr-seppia**

### CODE REVIEW

File: consensus/src/config.rs [hunk 1]
```diff
@@ -66,6 +66,11 @@ pub fn is_emergency_iter(iter: u8) -> bool {
     iter >= EMERGENCY_MODE_ITERATION_THRESHOLD
 }
 
+/// Returns if the next iteration generator needs to be excluded
+pub fn exclude_next_generator(iter: u8) -> bool {
```
Comment (goshawk-3) [inline consensus/src/config.rs:70]: Maybe `last_step_running`  should reuse this to avoid duplication?

Comment (goshawk-3) [inline consensus/src/config.rs:70]: `exlcude_next_generator` does not exclude anything?

Should be `is_last_iteration`?

**Review by goshawk-3 — CHANGES_REQUESTED** [review]

File: consensus/src/config.rs (hunk 1 above)
Comment (herr-seppia) [reply consensus/src/config.rs:70]: no, it's not last step, this applies to validation step too

Comment (goshawk-3) [reply consensus/src/config.rs:70]: I mean, you can reuse "exclude_next_generator/is_last_iteration" in `ExecutionCtx::last_step_running` that already exists.

Comment (herr-seppia) [reply consensus/src/config.rs:70]: I don't want to know if this is the last iteration.
I want to know it we should exclude the next iteration generator from the current committees.

They can be the same, I agree, but the concept is different.
Luckily the comment state this explicitly

Comment (herr-seppia) [reply consensus/src/config.rs:70]:
```rust
    /// Returns true if the last step of last iteration is currently running
    fn last_step_running(&self) -> bool {
        self.iteration == CONSENSUS_MAX_ITER - 1
            && self.step_name() == StepName::Ratification
    }
    ```
    
There is no `is_last_iteration`

Btw, if you mean to extract `self.iteration == CONSENSUS_MAX_ITER - 1` into a fn called `is_last_iteration` I agree.
I'll tackle it right after this PR

**Review by fed-franz — APPROVED** [review]

**Pull request merged**